packages/typespec-azure-portal-core/src/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface FilePath {
```

Comment:
@bterlson Do we have any mechanism for generating these types from the declarations in TypeSpec?